libtransact/src/handler/mod.rs

```diff
@@ -140,9 +140,9 @@ pub trait TransactionHandler: Send {
     /// transaction processor upon receiving a TpProcessRequest that the
     /// handler understands and will pass in the TpProcessRequest and an
     /// initialized instance of the Context type.
-    fn apply(
+    fn apply<'a>(
```

Comment:
This commit isn't necessary and is a breaking API change

Reply:
This change was initially required when the `KeyValueTransactionContext` was being created directly in the apply method. I was getting errors that the lifetimes were inconsistent since the txn ctx was expected to have `'a` lifetime but it is not assigned one in this method. However, I changed the context to be generic so this is no longer necessary in this impl.